src/react/DateTime/index.tsx

```diff
@@ -21,31 +21,36 @@ type DateTimeConstructorParameters = {
 }
 
 const DateTime = ({
-  value = new Date(),
```

Comment:
Why you want remove default value is new Date() ?
This changes the old behaviour of component